array.go

```diff
@@ -74,6 +74,7 @@ func (a *Array) Raw() []interface{} {
 // target should be one of these:
 //
 // - pointer to an empty interface
+//
 // - pointer to a slice of any type
```

Comment:
I guess here and below this changes are unintentional? I think you did not rebase on fresh master.

Reply:
No, it was intentional. without the extra gap, all list items will be formated in a single line. Should I change it back?

Reply:
I missed this detail for all the previous commits.

Reply:
Oh, I see. Nope, it was me, not you: 1623932e79bafe5a3e6d87623a14611851606b4d

Actually no need for empty lines, instead we need one more space. Instead of:

```
// - pointer to an empty interface
// - pointer to a slice of any type
```

we need:

```
//  - pointer to an empty interface
//  - pointer to a slice of any type
```

Would be awesome if you could fix it.

Reply:
Oh ok yea sure.